### PR TITLE
fix(client): Change colors

### PR DESCRIPTION
**Pull request by RakeshPotnuru**
Issue Number: #2431 

## PR Details

Changed background and hover colours for the support menu.

## PR Checklist
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Layout/SupportMenu.scss [hunk 1]
```diff
@@ -7,15 +7,15 @@ $menu-width: 180px;
   overflow: hidden;
   border-radius: var(--default-border-radius);
   min-width: $menu-width;
-  background-color: var(--menu-background-color);
+  background-color: #373D57;
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Layout/SupportMenu.scss:10]:
```suggestion
  background-color: var(--black10);
```

File: packages/amplication-client/src/Layout/SupportMenu.scss [hunk 2]
```diff
@@ -7,15 +7,15 @@ $menu-width: 180px;
...
   a {
     @include body2;
     display: block;
     padding: var(--default-spacing);
     color: var(--menu-item-color);
 
     &:hover {
-      background-color: var(--menu-item-active-background);
+      background-color: #444B66;
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Layout/SupportMenu.scss:18]:
```suggestion
      background-color: var(--black20);
```

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
Congrats on your first PR with Amplication.
Please see my suggestions 
Instead of using hardcoded color values, you should be using CSS variables

**Review by yuval-hazaz — APPROVED** [review]

**Comment by tupe12334** [comment]
Approved on staging

**Pull request merged**